harbour-osmscout.desktop

```diff
@@ -22,3 +22,7 @@ Name=OSM Scout
 # Remember to comment out the following line, if you do not want to use
 # a different app name in German locale (de).
 Name[hu]=OSMScout
+
+[X-HarbourBackup]
+BackupPathList=.local/share/harbour-osmscout/
```

Comment:
This path contains sqlite database with collections (`.local/share/harbour-osmscout/harbour-osmscout/storage.db`) and downloaded voices for navigation (`.local/share/harbour-osmscout/harbour-osmscout/voices/`). Voice package is small (~2 MiB per voice), still I am not sure if it should be part of the backup, because voices may be downloaded again...

Reply:
Collections should be part of backup. Voices are not neded.